provision/crio.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+source "${ENV_FILEPATH}"
+
+set -e
+
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+
+cat <<EOF > /etc/apt/sources.list.d/projectatomic-ubuntu-ppa-artful.list
+deb http://ppa.launchpad.net/projectatomic/ppa/ubuntu xenial main
+# deb-src http://ppa.launchpad.net/projectatomic/ppa/ubuntu artful main
+EOF
+
+sudo apt-get update
+sudo apt-get install cri-o-1.10 -y
```

Comment:
I would do, `sudo apt-get install -y cri-o-1.10` so it's easy to add new packages in the future.